tests/lmeval/test_lmeval.py

```diff
@@ -154,29 +154,46 @@ def _run_lm_eval(self):
             batch_size=self.lmeval.batch_size,
         )
 
-        metrics = results["results"][self.lmeval.task]
+        metrics: dict = results["results"][self.lmeval.task]
         for metric_key, expected_val in self.lmeval.metrics.items():
             # stderr metrics are only used as absolute tolerance
             # checks for actual values
             if "stderr" in metric_key:
                 continue
             actual_val = metrics.get(metric_key)
-            # If stderr is provided, use it as absolute tolerance
-            # Otherwise, default to a 5% relative tolerance
+            higher_is_better = results["higher_is_better"][self.lmeval.task].get(
```

Comment:
where is `higher_is_better` being defined?

Reply:
it's part of the lmeval data model, it's how they populate the table with up arrows or down arrows